content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,314 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
+destinations, aptly referred to as log forwarding. This page details how to
+configure log forwarding in Terraform Enterprise.
```

Comment:
I think we can omit the last sentence just because I think folks can get that from the page title. But! What I think we could write instead is maybe a sentence or two about why folks may want to do this? Something like "This is useful when you....... [x, y, z]"


```suggestion
Terraform Enterprise supports forwarding its logs to one or more external
destinations, a process called log forwarding. 
```

Reply:
My latest edits do not include the "why?". I'll write that up and push changes. Leaving this open until then.

Reply:
Pushed up changes that cover the "why?". Let me know what you think.

Reply:
Submitted the suggestion!